CHANGELOG.md

```diff
@@ -11,7 +11,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
```

Comment:
I think you need to add the version after line 8, right?
```markdown
 ## [0.4.0] - 2024-05-22
```

Reply:
I did! Well spotted